### DISCUSSION TITLE
Parameter values not getting shown in Simulation output

### DISCUSSION THREAD

**Discussion by Gautamvj** [Open-Systems-Pharmacology/Forum#1259] [How-Tos]
Dear All,
After performing population simulation, I wanted to compare my predicted parameter values with that of the reported observed values. For this when I open PK-analysis tab and view the parameter values I am not finding the values of few parameters, instead it is showing <NaN>. What can I do to get the values to be shown?
For your reference please see the attached image
![Vss   CL parameters values shown as _NaN](https://user-images.githubusercontent.com/107406825/180362868-bc97ed7d-b162-4a6d-a468-c491608ddb1a.JPG)

Regards,
Gautam.

**Comment by tobiasK2001** [comment]
Dear Gautam,
Sorry for the late reply. 
In my eyes there are several reasons for this and some of them are potential bugs or shortcomings of the PK analysis as discussed in more details here https://github.com/Open-Systems-Pharmacology/Forum/discussions/1065.
But reading through the comprehensive discussion will not bring the values you need... So here a short workaround: Create a box Wisker Analysis for you population simulation. 
Select the PK-Parameters of interest, e.g. all the missing and give some meaningful units: 
![image](https://user-images.githubusercontent.com/25585248/185961228-67272bd1-9ce8-47cd-94d6-ac7cce8cb6c2.png)
select them as outputs 
![image](https://user-images.githubusercontent.com/25585248/185961535-b797babb-0eb5-41f4-9aad-1fb8b253efd6.png)
By mouseover you can see the Median (50%) and the 5 to 95 % percentile of you population.
By right click on your Box whisker analysis you can export the results to excel as well.

Hope this might help!

Best, Tobias

**Comment by Yuri05** [comment]
@Gautamvj Some adjustments in the calculation of population PK parameters were made in the upcoming version 11.1 of the OSP, which might solve your problems.
Check the (preliminary) release notes for more details: https://github.com/Open-Systems-Pharmacology/Forum/discussions/1394
